test/factory/factory-server.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const Hapi = require('hapi')
+
+const port = Number(process.env.PORT) || 55155
+const options = {
+  connections: {
+    routes: {
+      cors: true
+    }
+  }
+}
+
+module.exports = (callback) => {
+  const http = new Hapi.Server(options)
```

Comment:
Do we need to bring in hapi? just using core-http should be enough for serving socket.io